real_time/arduino_src/lib_avr/servoreceiver/servoreceiver.cpp

```diff
@@ -173,18 +170,7 @@ Return the last read pulse width in microseconds.  This operation is atomic
 */
 unsigned long ServoReceiver::GetPulseWidth() 
 {
-    uint8_t oldSREG;
-    unsigned long temp;
-
-    oldSREG = SREG;
-
-    cli();
-    temp = rc_value_;
-
-    //Return the register to its original state rather than re-enable interrupts
-    SREG = oldSREG;
-
-    return temp;
+    return rc_value_;
```

Comment:
Why do we not do the interrupt disable here? Are we still guaranteed that this is atomic?